README.md

```diff
@@ -3,53 +3,94 @@
 [![chat][badge-chat]][badge-url-chat]
 
 ## Synopsis
-This middleware enables the use of [Crypt4GH][crypt4gh] files as inputs for tasks that are run in [TES][tes]
-implementations (e.g., [funnel][funnel], [TESK][tesk]).
+This proof-of-concept middleware enables the use of [Crypt4GH][crypt4gh] files as inputs for tasks that are 
+run in [TES][tes] implementations (e.g., [funnel][funnel], [TESK][tesk]).
+
 ## Description
+Currently, there are no implementations of TES that natively support the use of files encrypted with Crypt4GH.
+This middleware supports the use of Crypt4GH files by prepending the list of executors in a TES request with a
+decryption executor. This decryption executor decrypts the contents of any Crypt4GH files and places them in a volume
+so that subsequent executors can work on the decrypted contents.
+
+## Implementation Details
+
+### Middleware
+The middleware alters the initial TES request such that a decryption executor and a new volume (`/vol/crypt/`)are added 
```

Comment:
Leave a space between the closing parenthesis and "are" towards the end of the line